src/node.cc

```diff
@@ -4398,7 +4422,9 @@ int EmitExit(Environment* env) {
     Integer::New(env->isolate(), code)
   };
 
-  MakeCallback(env, process_object, "emit", arraysize(args), args);
+  MakeCallback(env->isolate(),
+               process_object, "emit", arraysize(args), args,
+               0).ToLocalChecked();
```

Comment:
Similar issue as with `beforeExit`, I'm not really sure what the appropriate action is.